java/Ice/context/README.md

```diff
@@ -0,0 +1,37 @@
+# Ice Context
+
+The Context demo shows the 3 different ways to set a request context in a client.
+
+A request context is a `dictionary<string, string>` carried by all requests. It is empty by default, and the application
+is free to set any entry in this dictionary.
+
+> [!NOTE]
+> The Ice protocol does not support response contexts, only request contexts.
+
+## Building the Project
+
+This project uses Gradle and the [Application plugin]. To build the client and server applications, run:
+
+```shell
+./gradlew build
```

Comment:
I checked out your PR and this fails. greeter works fine.

```
./gradlew build
Error: Could not find or load main class org.gradle.wrapper.GradleWrapperMain
Caused by: java.lang.ClassNotFoundException: org.gradle.wrapper.GradleWrapperMain
```

Reply:
The gradle wrapper was missing. Added.